### PR TITLE
Updated RasSearchCriteriaStatus to return a String array for ecosystm ras

### PR DESCRIPTION
**Pull request by KirbyKatcher**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/ras/RasSearchCriteriaStatus.java [hunk 1]
```diff
@@ -31,6 +34,14 @@ public boolean criteriaMatched(@NotNull TestStructure structure) {
    
 
    public List<TestRunLifecycleStatus> getStatuses() {
-       return this.statuses;
+      return this.statuses;
+   }
+
+   public String[] getStatusesAStrings() {
```
Comment (eamansour) [inline galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/ras/RasSearchCriteriaStatus.java:40]: Casing is a bit off - `getStatusesAsStrings()`

File: galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/ras/RasSearchCriteriaStatus.java [hunk 2]
```diff
@@ -31,6 +34,14 @@ public boolean criteriaMatched(@NotNull TestStructure structure) {
...
+      List<String> statuseStrings = new ArrayList<String>();
```
Comment (eamansour) [inline galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/ras/RasSearchCriteriaStatus.java:41]: Typo - `statuseStrings`

**Review by eamansour — APPROVED** [review]
Minor typos but approved assuming they'll be fixed.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-framework-pr-451-zwdgh for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**